producer/src/main/kotlin/streams/kafka/KafkaConfiguration.kt

```diff
@@ -28,6 +29,7 @@ data class KafkaConfiguration(val zookeeperConnect: String = "localhost:2181",
                               val transactionalId: String = StringUtils.EMPTY,
                               val lingerMs: Int = 1,
                               val topicDiscoveryPollingInterval: Long = TimeUnit.MINUTES.toMillis(5),
+                              val streamsLogCompactionStrategy: String = TopicConfig.CLEANUP_POLICY_DELETE,
```

Comment:
ah, there's the default.  It's fine to pick whichever default you wish (best to keep it the way it's been in the past for compatibility reasons).  See note earlier about naming / delete